modules/openssl/SCsub

```diff
@@ -671,7 +671,7 @@ if (env['builtin_openssl'] != 'no'):
     # Workaround for compilation error with GCC/Clang when -Werror is too greedy (GH-4517)
     import os
     import methods
-    if not (os.name == "nt" and os.getenv("VSINSTALLDIR") != None):  # not Windows and not MSVC
+    if (os.name != "nt" and os.getenv("VSINSTALLDIR") == None):  # not Windows and not MSVC
```

Comment:
The right fix would be to replace `VSINSTALLDIR` by `VCINSTALLDIR` a priori. Your fix will break compilation with MinGW on `nt`.

Reply:
came to me. you'v been faster to spot it than i RE-PR  :smiley:
fixed, rebased and tested again. Works ok. :thumbsup:

Reply:
Shouldn't this be `env.msvc` instead of `os.getenv("VCINSTALLDIR") != None` after #17194?

Reply:
Yeah but that's the 2.1 branch, where #17194 was not backported.